.yamato/global.metafile

```diff
@@ -0,0 +1,83 @@
+test_editors:
+  - version: 2019.4
+  - version: 2020.3
```

Comment:
We can drop 2019.4. End of life of 2020.3 is also coming soon.

Reply:
In Alembic's package.json, the minimum Unity version is still `"unity": "2019.4",`, because of that we cannot remove 2019.4 from CI, otherwise publishing workflow will fail.

Reply:
Good to know, thanks.